src/views/ManageList.jsx

```diff
@@ -17,9 +17,34 @@ export function ManageList({ listPath, userId }) {
 		setShareEmail(target.value);
 	};
 
+	const currentList = data.map((item) => {
+		return item.name
+			.toLowerCase()
+			.replace(/[!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~/\s]/g, '');
```

Comment:
praise: regular expressions are really intimidating to write at times, they way you both implemented it here looks like it'll catch any symbols that users might try to put in their list that might cause problems, nice!